sections/merge-accounts/common.ts

```diff
@@ -0,0 +1,227 @@
+import styled from 'styled-components';
```

Comment:
not relevant for this PR necessarily but we should try reusing more of the components globally within the staking app rather than having common components for each